controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/MetricsService.java

```diff
@@ -44,23 +48,79 @@ public DeploymentMetrics(double queriesPerSecond, double writesPerSecond,
         public double writeLatencyMillis() { return writeLatencyMillis; }
 
     }
-    
+
     class ApplicationMetrics {
 
         private final double queryServiceQuality;
         private final double writeServiceQuality;
-        
+
         public ApplicationMetrics(double queryServiceQuality, double writeServiceQuality) {
             this.queryServiceQuality = queryServiceQuality;
             this.writeServiceQuality = writeServiceQuality;
         }
-        
+
         /** Returns the quality of service for queries as a number between 1 (perfect) and 0 (none) */
         public double queryServiceQuality() { return queryServiceQuality; }
 
         /** Returns the quality of service for writes as a number between 1 (perfect) and 0 (none) */
         public double writeServiceQuality() { return writeServiceQuality; }
 
     }
-    
+
+    class CostMetrics {
+
+        private final double cpuUtil;
+        private final double memUtil;
+        private final double diskUtil;
+
+        public CostMetrics(double cpuUtil, double memUtil, double diskUtil) {
```

Comment:
Should we add diskBusy here while we are at it? Even if we don't have a good metric for it yet? 

Maybe also network util? netUtil?

Reply:
Yes, we can do that, but disk busy is really a trick one because we can't get it per docker container, we only have the docker host disk busy metric. Will that make sense to add in this scenario? 

I think we could wait a bit with network metrics and add them with the other metrics @yngveaasheim have in mind.